docs/variants/novacustom_ns5x_7x/overview.md

```diff
@@ -0,0 +1,18 @@
+# Overview
+
+* [Releases](releases.md) - list of all releases
```

Comment:
In section `Overview` we should use descriptions as in other refactored documentation, i. e. ASUS KGPE-D16.